### PR TITLE
v.1

### PR DESCRIPTION
**Pull request by Ramoxa**

### CODE REVIEW

File: stats/pom.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        <module>stats-server</module>
+        <module>stats-client</module>
+    </modules>
+
+    <properties>
+        <maven.compiler.source>21</maven.compiler.source>
+        <maven.compiler.target>21</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+    <dependencies>
```
Comment (litvitnik) [inline stats/pom.xml]: Обрати внимание, что dto точно не понадобится актуатор, а мы его подтягиваем для всех подмодулей stats

File: stats/stats-dto/pom.xml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <optional>true</optional>
+        </dependency>
+
+        <dependency>
```
Comment (litvitnik) [inline stats/stats-dto/pom.xml:29]: Нам не нужна jpa для dto

File: stats/pom.xml (hunk 1 above)
Comment (Ramoxa) [reply stats/pom.xml]: поправил

File: stats/stats-dto/pom.xml (hunk 2 above)
Comment (Ramoxa) [reply stats/stats-dto/pom.xml:29]: без нее только тогда у меня не запускается пакет..
rror: Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.13.0:compile (default-compile) on project stats-dto: Compilation failure: Compilation failure:
Error: /home/runner/work/java-explore-with-me/java-explore-with-me/stats/stats-dto/src/main/java/ru/practicum/ewm/dto/StatsDtoApp.java:[3,32] package org.springframework.boot does not exist
Error: /home/runner/work/java-explore-with-me/java-explore-with-me/stats/stats-dto/src/main/java/ru/practicum/ewm/dto/StatsDtoApp.java:[4,46] package org.springframework.boot.autoconfigure does not exist
Error: /home/runner/work/java-explore-with-me/java-explore-with-me/stats/stats-dto/src/main/java/ru/practicum/ewm/dto/StatsDtoApp.java:[6,2] cannot find symbol
Error: symbol: class SpringBootApplication
Error: -> [Help 1]
Error:
Error: To see the full stack trace of the errors, re-run Maven with the -e switch.
Error: Re-run Maven using the -X switch to enable full debug logging.
Error:
Error: For more information about the errors and possible solutions, please read the following articles:
Error: [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Error:
Error: After correcting the problems, you can resume the build with the command
Error: mvn -rf :stats-dto
Error: Process completed with exit code 1.

**Pull request merged**